index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>BMI Calculator</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <header></header>
+    <main>
+      <section class="bmi-container">
+        <div class="bmi-box">
+          <h1 class="heading">BMI Calculator</h1>
+        </div>
+        <div class="bmi-form-div">
+          <form class="bmi-form" action="">
+            <label for="Height">Your height:</label>
+            <input type="text" id="height" name="185cm" />
+            <label for="Weight">Your weight:</label>
+            <input type="text" id="weight" name="85kg" />
```

Comment:
You're missing some opportunities here:

First the `for` attribute in the `label` element helps for `focusing` the corresponding `input`. Therefor the `input` needs the `id` and the `name` as same as the `for` value.
Then, when you click on the Label with the mouse or touch it, the input is focused.

Read more about here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attributes

Second, you could provide a `placeholder` so the user knows what unit he or she has to input.

Reply:
ids and names on ```input```corrected. Added ```placeholder```on the missing fields.